.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: required
+
+language: java
+
+services:
+  - docker
+
+before_install:
+  - wget http://mirrors.ae-online.de/apache/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz
+  - tar xf apache-maven-3.3.9-bin.tar.gz
+  - export M2_HOME=$PWD/apache-maven-3.3.9
+  - export PATH=$M2_HOME/bin:$PATH
```

Comment:
As stated in the guide, they say it's 3.2. You can easily check it by running `mvn --version` instead of regular build goals.

As for advantanges/disadvantages. With the `wget` in the `.travis.yml` we only set it up for travis environment. With the wrapper we provide unified way of build the project everywhere. This also gives benefit to the users who want to hack on it as they don't have to bother about installing proper version of Maven. Everything is self-contained within the repo.